packages/reports/app/styles/navi-reports/components/favorite-item.less

```diff
@@ -5,8 +5,8 @@
 
 .favorite-item {
   color: @navi-dark-gray;
-}
-
-.favorite-item--active {
-  color: @favorite-gold;
+  
```

Comment:
Looks like some extra spaces `__` got included. If you're changing files and running `git commit` on your computer this should automatically be fixed up for you

Reply:
I've cleaned up the spaces. see the issue thread for other info.